packages/driver/src/codecs/registry.ts

```diff
@@ -261,6 +263,15 @@ export class CodecsRegistry {
           break;
         }
 
+        case CTYPE_RECORD: {
+          const els = frb.readUInt16();
+          for (let i = 0; i < els; i++) {
+            const elm_length = frb.readUInt32();
+            frb.discard(elm_length + 2);
+          }
+          break;
+        }
+
```

Comment:
I don't think this code will ever actually get executed, since for protocol v2+ the type descriptors are length prefixed, so we skip the whole typedesc in one go (line 113).

Reply:
Hm, good point, I'll double check